### ISSUE TITLE
ContextualMenu section headers are not read out when navigating into a section.

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1600]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Visible Focus indicator doesn't move
Expected: Focus indicator should follow focus

2. Can't invoke share submenu (first example). This works without JAWS
Expected: Right arrow or enter invokes submenus

3. Escape does not dismiss the menu
Expected: Escape dismisses menu and returns focus to origin or last place

**Comment by scriby** [comment]
1. Focus works properly when not using JAWS. Even though focus isn't moving, the "virtual cursor" is tracking properly, so this behavior may be acceptable given that JAWS users aren't looking at the screen.

**Comment by betrue-final-final** [comment]
Yeah, the tracking isn't a high pri issue. I thought we should record it though.

________________________________

**Comment by betrue-final-final** [comment]
All issues above are not repro in Narrator, but submenus are just called buttons, and should announce "haspopup" or be submenus. It's also odd that menu items are called buttons.

**Comment by betrue-final-final** [comment]
One more issue. The section headers are not read out when navigating into a section.

**Comment by petdun2519** [comment]
We need to remove the <button> component that is inside the menuitem <li>, the menuitem<li> is invokable in itself

**Comment by betrue-final-final** [comment]
All the issues are resolved excepting for the section headers bug.
